### PR TITLE
Locales: update ja_JP translations

### PR DESCRIPTION
**Pull request by knight-ryu12**
Fiddling with POEdit for like 2w, this finally done...
But there'll be some mistake that I couldn't take/catch.

this locale will be PR'd many time in future.

### CODE REVIEW

File: locales/ja_JP/pcsx2_Iconized.po [hunk 1]
```diff
@@ -1,36 +1,26 @@
-# SOME DESCRIPTIVE TITLE.
 # Copyright (C) YEAR PCSX2 Dev Team
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```
Comment (lightningterror) [inline locales/ja_JP/pcsx2_Iconized.po]: You could've update the copyright header while you were at it, I think some files had them so you can check them for a reference.

Comment (knight-ryu12) [reply locales/ja_JP/pcsx2_Iconized.po]: ahahaha. I will.

**Comment by pgert** [comment]
Next time, include the mo-files in ...\bin\Langs (if you want the translation to have effect).

**Comment by knight-ryu12** [comment]
oh yeah. I'll do that in separate PR today.

2018年11月29日(木) 8:16、pgert さん（notifications@github.com）のメッセージ:

**Pull request merged**